app/helpers/blacklight/maps_helper_override.rb

```diff
@@ -13,5 +13,15 @@ def document_path
       original_path = spotlight.exhibit_solr_document_path(exhibit_id: current_exhibit, id: token)
       original_path.sub(token, "{#{blacklight_config.document_model.unique_key}}")
     end
+
+    def index_map_data_attributes
```

Comment:
I don't suppose we could get an integration test for this?

And, let me see if I've got this right: for browse categories, we're redirecting the `search_url` parameter over to the catalog controller with the search parameters of this browse category,  because... the browse category url doesn't respond to the type of requests heatmaps is making (and... uh.. what requests are those?)

I might have a better suggestion based on the type of request, but I also wonder if we want to pass the browse category over to the search_exhibit_catalog_url and de-reference it there? It might be useful for bulk-actions-on-browse-categories, 🤷

Reply:
I've pushed up a commit with some tests around the integration piece.

> because... the browse category url doesn't respond to the type of requests heatmaps is making

Yep that seems part of it.. and also because naively in heatmaps we just use `request.url` to figure out what to do things on.

https://github.com/sul-dlss/blacklight_heatmaps/blob/master/app/helpers/blacklight/maps_helper.rb#L45

I'm not sure how that suggestion would work, but do we need to do something better upstream?

Reply:
The missing piece for me is... what do we do with `search_url`?

Reply:
We just pop on a `view=heatmaps` and clientside request away: https://github.com/sul-dlss/blacklight_heatmaps/blob/master/app/assets/javascripts/blacklight_heatmaps/viewers/index.js#L18

Reply:
Got it (searching for search_url didn't turn that up..). And browse categories don't response to `format=heatmaps`?

Reply:
Yeah.. getting a 

![Screen Shot 2021-03-10 at 11 42 32 AM](https://user-images.githubusercontent.com/1656824/110680359-bca56c00-8195-11eb-8518-0b760dd1899b.png)

Reply:
Ah, no, that's my copying the unreleased feature over instead of the 7.15-compatible version.